tpm2tools/template.go

```diff
@@ -47,6 +47,20 @@ func defaultRSADecrypt() *tpm2.RSAParams {
 	}
 }
 
+func defaultECCDecrypt() *tpm2.ECCParams {
+	return &tpm2.ECCParams{
+		Symmetric:  defaultSymScheme(),
+		CurveID: tpm2.CurveNISTP256,
+		Point: tpm2.ECPoint {
+			XRaw: make([]byte, 32),
+			YRaw: make([]byte, 32),
+		},
+		KDF: &tpm2.KDFScheme{
```

Comment:
Per the [documentation for `ECCParams`](https://godoc.org/github.com/google/go-tpm/tpm2#ECCParams), the `KDF` param is allowed to be `nil`, so you can just not set it here (`nil` is the default value).

If the `KDFScheme` is `nil`. [it still gets encoded as `AlgNull`](https://github.com/google/go-tpm/blob/ae6dd98980d43b6b1a71c9737490896ca6e6e9ed/tpm2/structures.go#L537)

Reply:
Done